### PR TITLE
fix: add config_cdn_uri param to DVCOptions initialize method

### PR DESCRIPTION
**Pull request by nsalamad**

### CODE REVIEW

File: lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb [hunk 1]
```diff
@@ -34,7 +35,7 @@ def initialize(
       end
 
       if config_polling_interval_ms < 1000
-        @logger.warn('config_polling_interval cannot be less than 1000ms, defaulting to 10000ms')
+        raise ArgumentError.new('config_polling_interval cannot be less than 1000ms, defaulting to 10000ms')
```
Comment (JamieSinn) [inline lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: This aborts the init though?  we're defaulting - not blocking the creation of the instance?

Comment (nsalamad) [reply lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: forgot to remove the line below - I'll fix that. But I think we should throw errors when invalid options are passed in. It's better to let the user know that they're doing something wrong than to default. Let me know what you think

Comment (JamieSinn) [reply lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: none of the other sdk's prevent creation of dvcoptions with out of bounds options - they just clamp them to the defaults.

Comment (nsalamad) [reply lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: nodejs throws but when you try to initialize the event queue: https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/eventQueue.ts#L60. I can move the checking in there if that's what you mean by 'none of the other sdk's prevent creation of dvcoptions'.  Also worth nothing that dotnet throws in DVCLocalOptions: https://github.com/DevCycleHQ/dotnet-server-sdk/blob/main/DevCycle.SDK.Server.Common/Model/Local/DVCLocalOptions.cs#L85

Comment (JamieSinn) [reply lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: hmm - interesting - we should add this as a test case in the test harness itself.

**Review by JamieSinn — APPROVED** [review]

File: lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb (hunk 1 above)
Comment (nsalamad) [reply lib/devcycle-ruby-server-sdk/localbucketing/dvc_options.rb:38]: agreed

**Pull request merged**